test/css/validate.js

```diff
@@ -0,0 +1,32 @@
+const assert = require('assert').strict;
+const { definitionSyntax } = require('css-tree');
+
+const css = require('@webref/css');
+
+const cssValues = [
+  { type: 'property', prop: 'properties', value: 'value' },
+  { type: 'extended property', prop: 'properties', value: 'newValue' },
```

Comment:
What is "newValue"? It looks to me like that doesn't appear in the input JSON files, so is this accidentally dead code?

Reply:
Aha, I think it should be "newValues", which is produced for things like https://drafts.csswg.org/css-flexbox-1/#flex-containers as a kind of partial value definition.

Reply:
Argh, indeed, that was a typo, it should have been `newValues`!

```suggestion
  { type: 'extended property', prop: 'properties', value: 'newValues' },
```